src/plenigo/models/ProductBase.php

```diff
@@ -175,6 +181,21 @@ public function setCustomAmount($condition)
         $this->customAmount = safe_boolval($condition);
     }
 
+    public function isSubscriptionRenewal()
+    {
+        return $this->subscriptionRenewal;
+    }
+
+    public function getSubscriptionRenewal()
```

Comment:
Actually there is a "reflection" type thing that adds "get" to the variable names afterwards, so for compatibility reasings I'm putting both